lib/broadside/gli/commands.rb

```diff
@@ -24,6 +24,7 @@ def add_command_flags(cmd)
 
 desc 'Bootstrap your service and task definition from the configured definition.'
 command :bootstrap do |bootstrap|
+  add_tag_flag(run)
```

Comment:
ugh, my bad. I fixed this in my local copy (like, literally the copy of the installed gem), and forgot to change it in both places.